source/ui/Components/Output/Console/ConsoleSettingsWindow.qml

```diff
@@ -1,159 +1,169 @@
 /*
- * C++ Assembler Interpreter
- * Copyright (C) 2016 Chair of Computer Architecture
- * at Technical University of Munich
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program. If not, see <http://www.gnu.org/licenses/>.
- */
+* C++ Assembler Interpreter
+* Copyright (C) 2016 Chair of Computer Architecture
+* at Technical University of Munich
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
 
 import QtQuick 2.6
 import QtQuick.Controls 1.4
 import QtQuick.Controls.Styles 1.4
 import QtQuick.Window 2.0
 import QtQuick.Layouts 1.3
+import "../../../Js/TextUtility.js" as TextUtility
+import Theme 1.0
 
-// Window for lightstrip settings.
+// Window for console settings.
 Window {
-    id: settingsWindowC
-    width: 400
-    height: 200
-    property int outputItemIndex: 2
-
-    title: "Console Settings"
-
-    // Refreshes the window's control contentItem.
-    function updateSettings() {
-        baseAddressTextField.text = outputComponent.getOutputItem(outputItemIndex)["baseAddress"];
-        var t = outputComponent.getOutputItem(outputItemIndex)["textMode"];
-        if(t === 0){
-            ab.checked = true;
+  id: settingsWindowConsole
+  width: Theme.console.settings.width
+  height: Theme.console.settings.height
+  property int outputItemIndex: 2
+
+  title: "Console Settings"
```

Comment:
I actually considered this, but I couldn't come up with a precise, yet short (enough) explanation. As we write a user manual, I think this is ok.
If you have concrete suggestions, feel free to add them/send them to me.